youtube_dl/extractor/telewebion.py

```diff
@@ -25,31 +25,26 @@ class TelewebionIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        secure_token = self._download_webpage(
-            'http://m.s2.telewebion.com/op/op?action=getSecurityToken', video_id)
-        episode_details = self._download_json(
-            'http://m.s2.telewebion.com/op/op', video_id,
-            query={'action': 'getEpisodeDetails', 'episode_id': video_id})
+        episode_details = self._download_json('https://gateway.telewebion.ir/kandoo/episode/getEpisodeDetail/?EpisodeId={}'.format(video_id), video_id)
+        episode_details = episode_details["body"]["queryEpisode"][0]
 
-        m3u8_url = 'http://m.s1.telewebion.com/smil/%s.m3u8?filepath=%s&m3u8=1&secure_token=%s' % (
-            video_id, episode_details['file_path'], secure_token)
+        channel_id = episode_details["channel"]["descriptor"]
+        episode_image_id = episode_details.get("image")
+        episode_image = "https://static.telewebion.com/episodeImages/{}/default".format(episode_image_id)
```

Comment:
1. "" -> '' here too
2. Don't crash if no `episode_image_id`
3. Ancient Pythons can't count here either
```suggestion
        channel_id = episode_details['channel']['descriptor']
        episode_image_id = episode_details.get('image')
        episode_image = 'https://static.telewebion.com/episodeImages/{0}/default'.format(episode_image_id) if episode_image_id else None
```